test/expect.js

```diff
@@ -622,6 +622,46 @@ describe('expect', function () {
     }, "blah: expected 1 to have property 'length'");
   });
 
+  it('above(n) (dates)', function(){
+    var now = new Date();
+    var oneSecondAgo = new Date(now.getTime() - 1);
+    var oneSecondAfter = new Date(now.getTime() + 1);
+
+    expect(now).to.be.above(oneSecondAgo);
+    expect(now).to.be.greaterThan(oneSecondAgo);
+    expect(now).to.not.be.above(now);
+    expect(now).to.not.be.above(oneSecondAfter);
+
+    err(function(){
+      expect(now).to.be.above(oneSecondAfter, 'blah');
+    }, "blah: expected " + now.toUTCString() + " to be above " + oneSecondAfter.toUTCString());
+
+    err(function(){
+      expect(10).to.not.be.above(6, 'blah');
+    }, "blah: expected 10 to be at most 6");
+
+    err(function () {
+      expect(now).to.have.length.above(4, 'blah');
+    }, "blah: expected " +  now.toUTCString() + " to have property 'length'");
+
+    err(function () {
+      expect([ 1, 2, 3 ]).to.have.length.above(now, 'blah');
+    }, "blah: the argument to above must be a number");
+
+    err(function () {
+      expect(null).to.be.above(now, 'blah');
+    }, "blah: expected null to be a number or a date");
+
+    err(function () {
+      expect(now).to.be.above(null, 'blah');
+    }, "blah: the argument to above must be a date");
+
+    // Is this right? Fails with the 'implementation frames error' (same as the 4 original ones in master)
+    err(function () {
+      expect(null).to.have.length.above(0, 'blah');
+    }, "Cannot read property 'length' of null");
```

Comment:
This test fails, along with 4 tests concerning instance checks (locally on clean `master`)
Any ideas?